### ISSUE TITLE
Saleor CLI 1.32.0 does not complete steps listed in template

### ISSUE THREAD

**Issue by PatricioNG** [saleor/saleor-app-template#177]
In the app template repository it's listed that the steps that will be completed are:
```
In this step, Saleor CLI will:
- clone this repository to the specified folder
- install dependencies
- ask you whether you'd like to install the app in the selected Saleor environment
- create .env file
- start the app in development mode
```

Yet following these steps does not produce that result, attached is the console output. After running `saleor app create test` I am prompted for app information but no repository is cloned, and a .env is not created.
![image](https://github.com/saleor/saleor-app-template/assets/46873633/f492baff-fea5-4469-8bb5-a750c051ce91)

Environment:
- Windows 11
- Node v20.3.1
- CLI installed via npm
- Saleor CLI 1.32.0

**Comment by lkostrowski** [comment]
@peelar can you please verify docs ?

**Comment by peelar** [comment]
Hello @PatricioNG 👋 

I was able to reproduce it. I passed it to our @saleor/devtools team. I will keep you posted.

**Comment by PatricioNG** [comment]
@peelar @lkostrowski Thanks both!

**Comment by zaiste** [comment]
The command `app create` was recently updated to `app template` to maintain consistency. The former is now used to trigger the mutaton of creating an `App`. (cf. https://github.com/saleor/saleor-cli/pull/645). 

I've updated the docs in this repo https://github.com/saleor/saleor-app-template/pull/178 cc @peelar 

@PatricioNG let me know if it works for you

**Comment by PatricioNG** [comment]
@zaiste yes it does work now to install the app, seems like there's additional differences between the docs here and the CLI however as all steps aren't documented. 

After installing step 5 `- Start the app in development mode` did not occur automatically, then after starting the app manually and running the `saleor app tunnel` command as indicated it throws an error:

![image](https://github.com/saleor/saleor-app-template/assets/46873633/d4fc132c-edf2-4ef2-bf4f-123f6154c8e8)

**Comment by lkostrowski** [comment]
@PatricioNG we will look into docs, meanwhile you can install ngrok to make it work:
https://ngrok.com/

You can use it directly (`ngrok http 3000`) or use CLI `saleor app tunnel` which wraps ngrok for you

You can also set up fully local development (no need for tunneling) - more here
https://docs.saleor.io/docs/3.x/developer/extending/apps/local-app-development
